core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -430,10 +430,10 @@ class ZooKeeperClient(connectString: String,
 
   // Visibility for testing
   private[zookeeper] def scheduleReinitialize(name: String, message: String, delayMs: Long): Unit = {
-    reinitializeScheduler.schedule(name, () => {
+    reinitializeScheduler.scheduleOnce(name, () => {
```

Comment:
How about calling ```schedule```? ```period``` and ```unit``` are equal to default value.

Reply:
Good point. Done.